scripts/apps/content-filters/views/filter-search.html

```diff
@@ -1,5 +1,10 @@
 <div ng-controller="FilterSearchCtrl">
-    <div class="panel-search">
+    <div class="split-content" >
```

Comment:
Please improve the formatting in this file and make it more obvious which node nests what, using indentation.